### PR TITLE
remove dcos dependency on dcoscli

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by jsancio** [comment]
@tamarrow We need to modify the `setup.py` list we do `cli/setup.py`: https://github.com/mesosphere/dcos-cli/blob/master/cli/setup.py#L81-L88

**Comment by jsancio** [comment]
Fixes #482

**Comment by tamarrow** [comment]
@jsancio won't it already be included with the `dcos` submodule?

**Comment by jsancio** [comment]
We need to add it to the `setup.py` for the `dcos` module: https://github.com/mesosphere/dcos-cli/blob/master/setup.py

**Comment by tamarrow** [comment]
@jsancio updated!

File: setup.py [hunk 1]
```diff
@@ -78,4 +78,13 @@
         'six>=1.9, <2.0',
         'toml>=0.9, <1.0',
     ],
+
+    # If there are data files included in your packages that need to be
+    # installed, specify them here.  If using Python 2.6 or less, then these
+    # have to be included in MANIFEST.in as well.
```
Comment (jsancio) [inline setup.py]: Remove this comment.

Comment (tamarrow) [inline setup.py]: done

**Comment by jsancio** [comment]
LGTM. :shipit: Let's make sure that we test this manually.

**Pull request merged**